lib/main.coffee

```diff
@@ -34,6 +34,12 @@ module.exports = new class Main
       type: "string"
       default: "deterministic"
       enum: ["deterministic"]
+    referTo:
+      title: "Custom rules based on"
+      description: "Set rules based on project name, or project full path"
+      type: "string"
+      default: "project name"
```

Comment:
This will change default behaviour for existing users. Might be better to keep default as "project path"

Reply:
So sorry I couldn't go ahead with this feature. Thank you very much for implementing it!